test/unit/controllers/EmailConfirmedControllerSpec.scala

```diff
@@ -121,6 +122,21 @@ class EmailConfirmedControllerSpec extends ControllerSpec with BeforeAndAfterEac
       redirectLocation(eventualResult).value should endWith("/manage-email-cds/cannot-change-email")
     }
 
+    "have a status of SEE_OTHER for show method when save email is failed with Error 400 or 500" in withAuthorisedUser() {
+      when(mockEmailCacheService.fetch(any())(any(), any())).thenReturn(Future.successful(Some(EmailDetails("abc@def.com", None))))
+
+
+      when(mockEmailVerificationService.isEmailVerified(any())(any[HeaderCarrier])).thenReturn(Future.successful(Some(true)))
+      when(mockUpdateVerifiedEmailService.updateVerifiedEmail(meq("abc@def.com"), meq("GB1234567890"))(any[HeaderCarrier]))
+        .thenReturn(Future.successful(None))
+
+      val eventualResult = controller.show(request)
+      status(eventualResult) shouldBe OK
+
+      //      redirectLocation(eventualResult).value should endWith("")
```

Comment:
comment line can be removed and test case description should be updated to `OK` response than `SEE_OTHER`

Reply:
Sorry, I forgot to remove this content line.